taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -140,7 +139,25 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
         }
       }
     });
-    setShownLanguages(allLanguageCodes);
+    // get shown languages from local storage if it exists else use all languages
+    try {
+      const lsShownLanguages = JSON.parse(
+        localStorage.getItem("shownLanguages")
+      );
+      if (lsShownLanguages) {
```

Comment:
please rename to `localStorageShownLanguages`
We do need to check that the data it contains is correct: 1) if is an array, 2) if is an array of strings, 3) filter all the items that are valid language codes. Only after those validations we can use the data